application/queue/deckids_to_userids.js

```diff
@@ -0,0 +1,9 @@
+/* global db */
```

Comment:
If the db object is global, then how (from where) should it be called?

Reply:
This is a mongo shell script file, can only run under the mongo shell. This file is not used directly, but via the `queue/deckids_import.sh` script.